src/bootstrap/setup.rs

```diff
@@ -68,11 +68,16 @@ pub fn setup(src_path: &Path, profile: Profile) {
 
     let path = cfg_file.unwrap_or_else(|| src_path.join("config.toml"));
     let settings = format!(
-        "# Includes one of the default files in src/bootstrap/defaults\n\
-    profile = \"{}\"\n",
+        "# Includes one of the default files in src/bootstrap/defaults
+    profile = \"{}\"
```

Comment:
nit: maybe use a raw string literal so you don't have to escape double quotes?
```suggestion
        r#"# Includes one of the default files in src/bootstrap/defaults
    profile = {}
```